### ISSUE TITLE
ImportError: cannot import name 'CallFunc'

### ISSUE THREAD

**Issue by patrickml** [PyCQA/prospector#260]
For some reason we have started getting this error -- we are getting it on multiple machines for multiple projects

```sh
Traceback (most recent call last):
  File "****/bin/prospector", line 7, in <module>
    from prospector.run import main
  File "****/lib/python3.6/site-packages/prospector/run.py", line 9, in <module>
    from prospector.config import configuration as cfg
  File "****/lib/python3.6/site-packages/prospector/config/__init__.py", line 7, in <module>
    from prospector.autodetect import autodetect_libraries
  File "****/lib/python3.6/site-packages/prospector/autodetect.py", line 4, in <module>
    from requirements_detector import find_requirements
  File "****/lib/python3.6/site-packages/requirements_detector/__init__.py", line 1, in <module>
    from requirements_detector.detect import find_requirements
  File "****/lib/python3.6/site-packages/requirements_detector/detect.py", line 5, in <module>
    from astroid import MANAGER, CallFunc, Name, Assign, Keyword, List, Tuple, Const, AssName
ImportError: cannot import name 'CallFunc'

```

**Comment by mattduck** [comment]
Just installed prospector for the first time and I'm hitting this issue.

Looks like it was caused by the 2.0 release of astroid a few days ago. I'm able to fix prospector with `pip install astroid==1.6.5`, which was the last 1.x release.

**Comment by patrickml** [comment]
Yeah you also have to change pylint as well

**Comment by carlio** [comment]
This should be fixed now as I released requirements-detector 0.6 earlier today which is where the problem was. Just `pip install -U prospector` or latest prospector will fix it.

I will leave this open while I see if there are any other compatability issues after the latest astroid/pylint releases.

**Comment by carlio** [comment]
Should all be fixed with v 1.0 just released but let me know if not.
